src/storage/src/hummock/sstable/builder.rs

```diff
@@ -135,6 +137,10 @@ pub struct SstableBuilder<W: SstableWriter, F: FilterBuilder> {
 
     epoch_set: BTreeSet<u64>,
     memory_limiter: Option<Arc<MemoryLimiter>>,
+
+    vnode_bitmap_mapping: HashMap<u32, BitmapBuilder>,
+    last_vnode: usize,
```

Comment:
How about using the vnode struct here instead of `usize`? https://github.com/risingwavelabs/risingwave/blob/b402d3b8a85e826def5b462691c6e16ff9da5721/src/common/src/hash/consistent_hash/vnode.rs#L34

Reply:
VirtualNode doesn't seem to offer any additional convenience, and we use BitmapBuilder directly without relying on VirtualNode.

The function signature is as follows

```rust
pub fn set(&mut self, n: usize, val: bool)
```

Reply:
Remove the `last_vnode`